src/main/java/com/google/swarm/tokenization/DLPTextToBigQueryStreamingV2.java

```diff
@@ -112,6 +113,9 @@ private static void runInspectAndDeidPipeline(
     if (options.getInputProviderType() == InputLocation.GCS && !usePubSub && !options.getProcessExistingFiles())
         throw new IllegalArgumentException("Either --processExistingFiles should be set to true or --gcsNotificationTopic should be provided");
 
+    if(options.getDataset()!= null && options.getOutputBucket()!=null)
+        throw new IllegalArgumentException("Please provide either BQ Dataset or GCS output bucket");
```

Comment:
Can the input bucket and the output bucket be the same?

Reply:
Yes both can be same, but the customer needs to pass the filePattern carefully.